app_feup/analysis_options.yaml

```diff
@@ -2,4 +2,27 @@ linter:
   rules:
     - annotate_overrides
     - hash_and_equals
-    - prefer_is_not_empty
\ No newline at end of file
+    - prefer_is_not_empty
+    - avoid_empty_else
+    - avoid_print
+    - avoid_relative_lib_imports
+    - empty_statements
+    - no_duplicate_case_values
+    - await_only_futures
+    - camel_case_types
+    - constant_identifier_names
+    - file_names
+    - implementation_imports
+    - lines_longer_than_80_chars
+    - non_constant_identifier_names
+    - prefer_equal_for_default_values
+    - prefer_final_fields
+    - prefer_final_locals
+    - package_names
+
```

Comment:
These seem appropriate. I suggest also adding the following
- valid_regexps
- curly_braces_in_flow_control_structures
- prefer_is_empty
- prefer_single_quotes
- recursive_getters
- unnecessary_new

What do you think?

Reply:
They all seem valid! Going to add them now.